Decorator/Program.cs

```diff
@@ -7,8 +7,8 @@ class Program
     {
         static void Main(string[] args)
         {
-            string text = "FUCK YOU https://forums.envato.com/t/check-if-string-is-a-url/75760/2";
-            Console.WriteLine(new ReplaceBadWords(new RemoveCapsLockRage(new AntiSpam())).Filter(text));
+            TextFilter textFilter = new EmojiDecorator(new RemoveCapsLockRage());
+            string text = textFilter.Filter("BAZINGA :-*");
```

Comment:
Here you create a decorator and passes another decorator in it. Please use an example that I sent you:
"You have text provider that have methods InputText (save some text inside) and OutputText(outputs to console). Write decorator for the output to replace text smiles to emoji"

Reply:
done